EDQueue.podspec

```diff
@@ -1,14 +1,14 @@
 Pod::Spec.new do |s|
   s.name         = 'EDQueue'
-  s.version      = '0.7.1'
+  s.version      = '0.7.3'
   s.license      = 'MIT'
   s.summary      = 'A persistent background job queue for iOS.'
-  s.homepage     = 'https://github.com/thisandagain/queue'
-  s.authors      = {'Andrew Sliwinski' => 'andrewsliwinski@acm.org', 'Francois Lambert' => 'flambert@mirego.com'}
-  s.source       = { :git => 'https://github.com/thisandagain/queue.git', :tag => 'v0.7.1' }
-  s.platform     = :ios, '5.0'
+  s.homepage     = 'https://github.com/gelosi/queue'
```

Comment:
I've set it to mine now to use it already, but if you will accept the changes in the end (I'm still on it) I'd be happy to keep original link to repo.